README.md

```diff
@@ -52,78 +48,55 @@ $ docker run -e "DB_HOST=55.55.55.1" -e "DB_PORT=1234" <image name>[:tags]
 
 Kafka data is saved to `/kafkadata` in the docker container. To persist the data, create a volume and mount it
 
-## Environmental variables
-
-**JDBC Driver**
-
-|  Name                        | Purpose                          | Default        |
-| ---------------------------- | -------------------------------- | -------------- |
-| JDBC_DRIVER_URI              |                                  | https://jdbc.postgresql.org/download/postgresql-42.1.1.jre7.jar |
-| JDBC_DRIVER_URI_USERNAME     |                                  |                |
-| JDBC_DRIVER_URI_PASSWORD     |                                  |                |
-
+### Environment variables
 
-**Global database variables**
+**Database connection**
 
 |  Name                        | Purpose                          | Default        |
 | ---------------------------- | -------------------------------- | -------------- |
-| DB_USERNAME                  |                                  | postgres       |
-| DB_PASSWORD | | postgres |
-| DB_HOST | | localhost |
-| DB_PORT | | 5432 |
-| DB_NAME | | postgres |
+| JDBC_DRIVER_URI              | Download (curl) the specified database driver.  If no driver is specified, you must bake a driver into your image.  See *Constructing Your Image* for more information on baking a driver in. | https://jdbc.postgresql.org/download/postgresql-42.1.1.jre7.jar |
```

Comment:
If no driver is specified, you must bake a driver into your image.  See *Constructing Your Image* for more information on baking a driver in.==>
If you do not specify a driver, you must embed the driver into your Docker image (is it a docker image we're talking about or the JDBC URI???)